homeassistant/components/fastdotcom/sensor.py

```diff
@@ -8,21 +8,20 @@
     SensorEntity,
     SensorStateClass,
 )
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import UnitOfDataRate
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.restore_state import RestoreEntity
-from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
-from . import DATA_UPDATED, DOMAIN as FASTDOTCOM_DOMAIN
+from .const import DATA_UPDATED, DOMAIN as FASTDOTCOM_DOMAIN
 
 
-async def async_setup_platform(
+async def async_setup_entry(
```

Comment:
I think if we remove the `setup_platform()` now we will get a separate repair issue about the YAML.
Can you test this if that's the case and restore an empty `setup_platform()` in that case?

Reply:
My test didn't showed any weird behavior, yet I just leave in an empty function just to be 100% sure. Hence, you hinted on this, so I suppose there are use cases it might conflicts randonmly. An empty function might prevent this.

Reply:
Pylint went haywire on an empty function. I inspired myself of the [NextBus](https://github.com/home-assistant/core/blob/2531b0bc09dba8d85243e8b7a4942840ef7b64d5/homeassistant/components/nextbus/sensor.py#L41), to include the create_issue here.
I recommend removing the create issue from the config flow.